internal/servicedeployer/_static/Dockerfile.terraform_deployer

```diff
@@ -1,19 +1,22 @@
-FROM --platform=linux/amd64 ubuntu:20.04
-ENV GCLOUD_SDK_VERSION 370.0.0-0
-ENV TERRAFORM_VERSION 1.1.4
+FROM --platform=linux/amd64 ubuntu:24.04
+ENV GCLOUD_SDK_VERSION=467.0.0-0
```

Comment:
Currently, there are no packages creating resources in google cloud, but it looks good to update this version too.